### DISCUSSION TITLE
Comments: Implementing Guest Authentication with AWS Amplify

### DISCUSSION THREAD

**Discussion by javamonn** [javamonn/literal#102] [General]
> A technical walkthrough of Literal's guest authentication implementation.

Read the text [here](https://literal.io/writing/implementing-guest-authentication-with-amplify-cognito-appsync).

**Comment by paulsingh** [comment]
Ok, first of all, _your walkthrough is incredible_. Thank you for putting it together. 👏🏼

I stumbled across your post today and am *slowly* piecing together your instructions. 

I don’t suppose you’ve tackled the [GraphQL Transformer v1 to v2 migration](https://docs.amplify.aws/cli/migration/transformer-migration/) around this same topic?
